examples/xslt-transformer/xslt_transformation.bal

```diff
@@ -0,0 +1,64 @@
+import ballerina/io;
```

Comment:
filename should be xslt-transformation.bal right?
Shall we change other occurrences as well?

Reply:
No, folder name should be in hyphen but underscore should be in filename.